### PR TITLE
New domain names from www.co.pl

### PR DESCRIPTION
**Pull request by wwwcopl**
More details: http://www.co.pl/dokumenty/obslugiwane-nazwy-domen/

### CODE REVIEW

**Comment by wwwcopl** [comment]
Any news about the merge to master? Any date?

**Comment by weppos** [comment]
Please add the necessary validation records, as requested by https://github.com/publicsuffix/list/wiki/Guidelines

Moreover, please explain the reason of the request.

As a side note, I'd like to inform you that to protect the consumption of the PSL, we are considering to limit the number of per-entity private domains. At the time being, I would be ok to include the domains in this request. However, we may not be able to accommodate the request if the number should continue to increase in the future.

**Comment by wwwcopl** [comment]
**Domain verification**
At the beginning I would like to mention that currently all domains, which are included in the commit https://github.com/publicsuffix/list/pull/142 have configured DNS TXT _psl entry with a link to our commit and are prepared for verification. Here you can find the full list of our domains: http://www.co.pl/dokumenty/obslugiwane-nazwy-domen/ .

**Reason**
For many years, we enable our customers to register domain tld co.pl. This is our first custom TLD - already added to the list PublicSuffix. Some time ago we decided to expand our offer with new alternative regional TLDs. We currently have approximately 180 new domains, which are used to register subdomains for our customers. We would like these domains to be treated as TLDs and ensure that customers can use them freely (for example on domain parkings like ParkingCrew).

**Comment by wwwcopl** [comment]
Hello,

any news? I added validation records and wrote the reason. 
On 2016-11-19 17:20:41, Simone Carletti <notifications@github.com> napisał:
Please add the necessary validation records, as requested by https://github.com/publicsuffix/list/wiki/Guidelines [https://github.com/publicsuffix/list/wiki/Guidelines]
Moreover, please explain the reason of the request.
As a side note, I'd like to inform you that to protect the consumption of the PSL, we are considering to limit the number of per-entity private domains. At the time being, I would be ok to include the domains in this request. However, we may not be able to accommodate the request if the number should continue to increase in the future.

**Comment by wwwcopl** [comment]
Hello,

any news? I added validation records and wrote the reason.
On 2016-11-19 17:20:41, Simone Carletti <notifications@github.com> napisał:
Please add the necessary validation records, as requested by https://github.com/publicsuffix/list/wiki/Guidelines [https://github.com/publicsuffix/list/wiki/Guidelines]
Moreover, please explain the reason of the request.
As a side note, I'd like to inform you that to protect the consumption of the PSL, we are considering to limit the number of per-entity private domains. At the time being, I would be ok to include the domains in this request. However, we may not be able to accommodate the request if the number should continue to increase in the future.

File: public_suffix_list.dat [hunk 1]
```diff
@@ -11224,6 +11224,185 @@ info.at
 
 // Michau Enterprises Limited : http://www.co.pl/
 co.pl
```
Comment (sleevi) [inline public_suffix_list.dat]: Could you either grant edit access to the maintainers for this PR, or sort this appropriately?

I've validated the _PSL record is in place for all of these domains, and that it matches http://www.co.pl/wp-content/uploads/2014/09/listadomen.txt as the list of domains.

**Comment by sleevi** [comment]
Closing due to no feedback.

**Pull request closed without merging**